README.md

```diff
@@ -1,35 +1,466 @@
+    - [Sending Events With Payloads](#sending-events-with-payloads)
+    - [Send User Properties](#send-user-properties)
+    - [Sending Virtual Economy Purchases](#sending-virtual-economy-purchases)
+    - [Sending Virtual Economy Currency Given](#sending-virtual-economy-currency-given)
+    - [Sending IAP Events and IAP Validation](#sending-iap-events-and-iap-validation)
+  - [In-app Messaging](#in-app-messaging)
+    - [In-app Messaging Deeplinks](#in-app-messaging-deeplinks)
+  - [iOS Push Notifications](#ios-push-notifications)
+    - [Creating and uploading your iOS Cert](#creating-and-uploading-your-ios-cert)
+    - [Provisioning your app](#provisioning-your-app)
+  - [Android Push Notifications](#android-push-notifications)
+    - [Using a custom `GcmReceiver`](#using-a-custom-gcmreceiver)
+    - [Custom `MainActivity`](#custom-mainactivity)
+    - [No custom `MainActivity`](#no-custom-mainactivity)
+    - [Initializing the SDK](#initializing-the-sdk)
+    - [Advanced Android Push](#advanced-android-push)
+  - [Integrating Resource A/B Testing](#integrating-resource-ab-testing)
+  - [Testing your integration](#testing-your-integration)
+- [Upgrade Instructions](#upgrade-instructions)
+- [How to run the demo](#how-to-run-the-demo)
+- [How to build the SDK](#how-to-build-the-sdk)
+- [Contributing](#contributing)
+- [License](#license)
+
 Getting started
----------------
-Have a look at the quick integration guide at http://docs.swrve.com/developer-documentation/37926233/
+=
 
 Requirements
-------------
+-
+Swrve supports Unity version 4.0.0 or later on Android and iOS only. **Swrve Unity SDK is not currently supported on Unity Web Player or Windows 8.**
+
 ### Unity 4.0.0+
 The SDK needs a Unity version higher than 4.0.0 to be able to compile.
 
 ### Unity 5+ Personal or Unity 4.0.0+ Pro
 Some of the features like push notifications require the use of native plugins. Native plugins are now available in Unity Personal 5+ or previvous versions of Unity Pro.
 
+Installation Instructions
+=
+1. Download the [latest release](https://github.com/Swrve/swrve-unity-sdk/releases/latest)
+
+2. Unzip the Unity SDK, import the `Swrve.unityPackage` into your project and add the `SwrvePrefab` component to your scene.
+
+  ![Import swrve.unityPackage](/docs/images/name12.jpg)
+
+  ![Add Prefab to Scene](/docs/images/name13.jpg)
+
+3. Initialize the Swrve Component
+
+  There are two methods to initialize the SDK component. Automatic initialization and manual initialization. In almost all cases, you should you manual initialization and ensure that the "initialise on start" property is not set in the Swrve prefab.
+
+  Set the `app_id` and `api_key` fields of the component to your Swrve App ID and Swrve API Key.
+
+  ```
+  // By default, Swrve stores all customer data and content in our US data center. 
+  //SwrveComponent.Instance.Config.SelectedStack = Swrve.Stack.EU;
+
+  SwrveComponent.Instance.Init(<app_id>, "<api_key>");
```

Comment:
would something like this work for you? 

  #if PRODUCTION_BUILD
  SwrveComponent.Instance.Init(<production_app_id>, "<production_api_key>");
  #else
  SwrveComponent.Instance.Init(<sandbox_app_id>, "<sandbox_api_key>");
  #endif //PRODUCTION BUILD